pkgs/by-name/ja/jawiki-all-titles-in-ns0/package.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, fetchurl
+, stdenvNoCC
+}:
+
+let
+date = "20240520";
+jawiki = fetchurl {
+  url ="https://dumps.wikimedia.org/jawiki/${date}/jawiki-${date}-all-titles-in-ns0.gz";
```

Comment:
<https://dumps.wikimedia.org/jawiki/> only keeps data for three months so this will have to be mirrored.